docs/source/delta-row-tracking.md

```diff
@@ -0,0 +1,113 @@
+orphan: 1
+---
+
+# Use row tracking for Delta tables
+
+Row tracking allows <Delta> to track row-level lineage in a <Delta> table. When enabled on a <Delta> table, row tracking adds two new metadata fields to the table:
+
+- **Row IDs** provide rows with an identifier that is unique within the table. A row keeps the same ID whenever it is modified using a `MERGE` or `UPDATE` statement.
+
+- **Row commit versions** record the last version of the table in which the row was modified. A row is assigned a new version whenever it is modified using a `MERGE` or `UPDATE` statement.
+
+.. note:: This feature is available in <Delta> 3.2.0 and above. This feature is in experimental support mode with [_](#limitations).
+
+## Enable row tracking
+
+You must explicitly enable row tracking using one of the following methods:
+
+- **New table**: Set the table property `delta.enableRowTracking = true` in the `CREATE TABLE` command.
+
+  ```sql
+  -- Create an empty table
+  CREATE TABLE student (id INT, name STRING, age INT) TBLPROPERTIES ('delta.enableRowTracking' = 'true');
+
+  -- Using a CTAS statement
+  CREATE TABLE course_new
+  TBLPROPERTIES ('delta.enableRowTracking' = 'true')
+  AS SELECT * FROM course_old;
+
+  -- Using a LIKE statement to copy configuration
+  CREATE TABLE graduate LIKE student;
+
+  -- Using a CLONE statement to copy configuration
+  CREATE TABLE graduate CLONE student;
+```
+
+- **Existing table**: Set the table property `'delta.enableRowTracking' = 'true'` in the `ALTER TABLE` command.
+
+  ```sql
+  ALTER TABLE grade SET TBLPROPERTIES ('delta.enableRowTracking' = 'true');
+  ```
+
+- **All new tables**: Set the configuration `spark.databricks.delta.properties.defaults.enableRowTracking = true` for the current session in the `SET` command.
+
+  .. code-language-tabs::
+    ```sql
+    SET spark.databricks.delta.properties.defaults.enableRowTracking = true;
+    ```
+
+    ```python
+    spark.conf.set("spark.databricks.delta.properties.defaults.enableRowTracking", True)
+    ```
+
+    ```scala
+    spark.conf.set("spark.databricks.delta.properties.defaults.enableRowTracking", true)
+    ```
+
+.. important:: Because cloning a <Delta> table creates a separate history, the row ids and row commit versions on cloned tables do not match that of the original table.
+
+.. important:: Enabling row tracking on existing table will automatically assign row ids and row commit versions to all existing rows in the table. This process may cause multiple new versions of the table to be created and may take a long time.
+
+.. warning:: Tables created with row tracking enabled have the row tracking <Delta> table feature enabled at creation and use <Delta> writer version 7. Table protocol versions cannot be downgraded, and tables with row tracking enabled are not writeable by <Delta> clients that do not support all enabled <Delta> writer protocol table features. See [_](/versioning.md).
```

Comment:
I think this is important enough to be pulled as the first element in "Enable row tracking" section